src/App.jsx

```diff
@@ -1,17 +1,19 @@
-import { useState } from 'react'
 import './App.css';
-import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
+import { useState } from 'react';
 
 function App() {
+  //create a new state var: newTodo, setNewTodo
+  const [newTodo, setNewTodo] = useState(" ");
 
   return(
-  <div>
-    <h1>TODO LIST</h1>
-    <AddTodoForm />
-    <TodoList />
-  </div>
+    <div>
+      <h1>TODO LIST</h1>
+      <AddTodoForm setNewTodo={setNewTodo} />
```

Comment:
As the instructions indicate you should pass the setNewTodo setter function as a onAddTodo function, then in AddTodoForm we should destructure that onAddTodo prop and use it there to pass the todoTitle